test/metabase/task/analyze_queries_test.clj

```diff
@@ -0,0 +1,68 @@
+(ns metabase.task.analyze-queries-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase.lib.core :as lib]
+   [metabase.lib.metadata :as lib.metadata]
+   [metabase.lib.metadata.jvm :as lib.metadata.jvm]
+   [metabase.models :refer [Card]]
+   [metabase.query-analysis :as query-analysis]
+   [metabase.task.analyze-queries :as task.analyze-queries]
+   [metabase.test :as mt]
+   [metabase.util :as u]
+   [metabase.util.queue :as queue]
+   [toucan2.core :as t2]))
+
+(deftest analyzer-loop-test
+  (let [metadata-provider (lib.metadata.jvm/application-database-metadata-provider (mt/id))
+        venues            (lib.metadata/table metadata-provider (mt/id :venues))
+        venues-name       (lib.metadata/field metadata-provider (mt/id :venues :name))
+        mlv2-query        (-> (lib/query metadata-provider venues)
+                              (lib/aggregate (lib/distinct venues-name)))]
+    (mt/with-temp [Card c1   {:query_type    "native"
```

Comment:
It'd be nice to DRY these fixtures up with the backfill / sweeper test as well. Will probably do that when I rework that task.